spec/registrar/register_object_spec.rb

```diff
@@ -25,12 +25,21 @@
       params = {}
       response = double('net http response', to_hash: { 'Status' => ['200 OK'] }, code: 200, body: 'not_valid_druid')
       registrar = Was::Registrar::RegisterObject.new
-      allow(RestClient).to receive(:post).and_return(response)
-      expect { registrar.register_object_using_web_service(params) }.to raise_error('Error in registring the object. Not valid druid returned not_valid_druid')
+      expect(RestClient).to receive(:post).and_return(response)
+      expect { registrar.register_object_using_web_service(params) }.to raise_error(RuntimeError)
```

Comment:
like the idea that we test that the error message makes sense for the error thrown ...